docs/learn/archive/learn-treasury.md

```diff
@@ -74,9 +74,6 @@ The Treasury is funded from different sources:
    inflation to go to the Treasury. In other words, if 50% of all tokens are staked, then 100% of
    the inflation goes to the validators as reward. If the staking rate is greater than or less than
    50%, then the validators will receive less, with the remainder going to the Treasury.
-4. Parathreads: [Parathreads](../learn-parathreads.md) participate in a per-block auction for block
```

Comment:
Have to check if on-demand parachain fees goes to the treasury or it is burned. Can be updated through a followup PR.